LLMonFHIR.xcodeproj/project.pbxproj

```diff
@@ -599,10 +603,10 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = "LLMonFHIR/Supporting Files/LLMonFHIR.entitlements";
 				CODE_SIGN_IDENTITY = "Apple Development";
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "";
-				DEVELOPMENT_TEAM = 637867499T;
+				DEVELOPMENT_TEAM = "";
```

Comment:
Please ensure that you have enabled "Automatic Code Signing" + selected "None" as your development team in the Test Configuration for all targets:
<img width="1178" alt="Screenshot 2023-07-12 at 6 33 03 PM" src="https://github.com/StanfordBDHG/LLMonFHIR/assets/28656495/6ce3ed28-481a-4fc5-97b5-210057cfcf5c">